ceph.spec.in

```diff
@@ -359,15 +359,21 @@ Requires:       python%{_python_buildid}-pecan
 Requires:       python%{_python_buildid}-werkzeug
 Requires:       pyOpenSSL%{_python_buildid}
 %endif
+%if 0%{?fedora}
+Requires:	python%{_python_buildid}-bcrypt
+%endif
+%if 0%{?rhel}
+Requires:	py-bcrypt
+%endif
```

Comment:
I would rather see these fedora and rhel conditionals inside the `%if 0%{?fedora} || 0%{?rhel}` block, but that's just me.

Reply:
That was my initial plan, but the package names differ between fedora and rhel

Reply:
@rjfd Right, just move your two conditionals into the preceding conditional.

Reply:
But opinions may differ on this - I can live with it either way.

Reply:
ah, that makes sense

Reply:
Do these kind of conditionals work on any RPM-based distribution? It was my understanding the these macros (e.g. `0%{rhel}` are only available when building the RPM in the openSUSE Build Service.